internal/gatewayapi/testdata/httproute-with-header-filter-duplicate-adds.out.yaml

```diff
@@ -81,10 +81,6 @@ httpRoutes:
       conditions:
       - type: Accepted
         status: "True"
```

Comment:
ditto

Reply:
@skriss commit e14991a updates the PR to only set the "UnsupportedValue" reason when `Accepted=false`.